pangeanetwork.py

```diff
@@ -352,6 +352,28 @@ def loans():
   results = {'data': data}
   return Response(json.dumps(results), mimetype='application/json')
 
+@app.route('/loan/<loanid>', methods=['GET'])
+def loan(loanid):
+  loans = Loan.query.all()
+  data = []
+  loanid = int(loanid)
+  for loan in loans:
```

Comment:
for the loan lookup we can still use the `.query.filter` instead of the `.query.all` so we can fetch the loan we're looking for directly instead of looping through them. It should be similar to the user filter so something like `Loan.query.filter(loanid == Loan.id)`